### PR TITLE
Add ability do not wait deployment to complete. Just trigger master

### LINKED ISSUE #2757: Support async stack deployments

### PR DESCRIPTION
**Pull request by dzubchik**
This fixes #2757

### CODE REVIEW

File: cli/lib/kontena/cli/stacks/deploy_command.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ class DeployCommand < Kontena::Command
 
     parameter "NAME", "Stack name"
 
+    option '--async', :flag, 'Do not wait service deployment', default: false
```
Comment (jnummelin) [inline cli/lib/kontena/cli/stacks/deploy_command.rb]: This is human-facing "interface", maybe `--no-wait` would make more sense?

File: cli/lib/kontena/cli/stacks/deploy_command.rb [hunk 2]
```diff
@@ -21,7 +23,9 @@ def execute
       spinner "Waiting for deployment to start" do
         wait_for_deployment_to_start(deployment)
```
Comment (jnummelin) [inline cli/lib/kontena/cli/stacks/deploy_command.rb]: I don't see why we would want to wait for the deployment to start if we're not gonna wait for it to finish either. So I'd wrap this also within  `if !async`

File: cli/lib/kontena/cli/stacks/deploy_command.rb [hunk 3]
```diff
@@ -21,7 +23,9 @@ def execute
...
       end
-      wait_for_deploy_to_finish(deployment)
+      if !async?
```
Comment (jnummelin) [inline cli/lib/kontena/cli/stacks/deploy_command.rb]: I think we're using more `unless self.async?` style for these kind of checks, reads a bit better :)

File: cli/lib/kontena/cli/stacks/deploy_command.rb [hunk 4]
```diff
@@ -18,10 +20,12 @@ def execute
       spinner "Triggering deployment of stack #{pastel.cyan(name)}" do
         deployment = deploy_stack(name)
       end
-      spinner "Waiting for deployment to start" do
-        wait_for_deployment_to_start(deployment)
+      unless :'no-wait'?
```
Comment (SpComb) [inline cli/lib/kontena/cli/stacks/deploy_command.rb]: `no_wait?`

**Comment by dzubchik** [comment]
@jnummelin , updated PR according to comments. I've done --no-wait by analogy from upgrade command. Is it ok?

**Pull request merged**